cli/init.ts

```diff
@@ -0,0 +1,97 @@
+import { Command } from "../deps/cliffy.ts"
+import * as fs from "../deps/std/fs.ts"
+import { parse } from "../deps/std/jsonc.ts"
+import * as path from "../deps/std/path.ts"
+import { detectVersion } from "../server/detectVersion.ts"
+
+export const init = new Command()
+  .description("Configure Capi in the current working directory")
+  .stopEarly()
+  .example("Configure Capi in the current directory", "capi init")
+  .action(runInit)
+
+async function runInit() {
+  const packageJsonPath = path.join(Deno.cwd(), "package.json")
+  if (await fs.exists(packageJsonPath)) return await runInitNode(packageJsonPath)
+  let denoJsonPath = path.join(Deno.cwd(), "deno.json")
+  if (await fs.exists(denoJsonPath)) return await runInitDeno(denoJsonPath)
+  denoJsonPath += "c"
+  if (await fs.exists(denoJsonPath)) return await runInitDeno(denoJsonPath)
+  throw new Error("Could not find neither a `package.json` nor `deno.json`/`deno.jsonc`.")
+}
+
+async function runInitNode(packageJsonPath: string) {
+  const packageJson = JSON.parse(await Deno.readTextFile(packageJsonPath))
+  assertManifest(packageJson)
+  const isTs = confirm("Are you using TypeScript?")
+  const devDependencies = packageJson.devDependencies ??= {}
+  if (isTs && !devDependencies["ts-node"]) {
+    const addTsNode = confirm(
+      `To define your net specs in TypeScript, we'll need to install the \`ts-node\` loader. Can we add it to your \`package.json\`?`,
+    )
+    if (addTsNode) {
+      devDependencies["ts-node"] = "*"
+    }
+  }
+  const dependencies = packageJson.dependencies ??= {}
+  dependencies.capi = detectVersion() ?? "*"
+  const scripts = packageJson.scripts ??= {}
+  scripts["capi:sync"] = "capi sync node"
+  scripts["capi:serve"] = "capi serve"
+  Deno.writeTextFileSync(packageJsonPath, JSON.stringify(packageJsonPath, null, 2))
```

Comment:
it should be `JSON.stringify(packageJson, null, 2)` with the updated dependencies